Havit/Havit/Screens/CategoryContents/View/CategoryContentsViewController.swift

```diff
@@ -7,22 +7,187 @@
 
 import UIKit
 
-class CategoryContentsViewController: UIViewController {
+import SnapKit
+import RxSwift
+
+class CategoryContentsViewController: BaseViewController {
+    var searchController: UISearchController = {
+        let searchController = UISearchController(searchResultsController: nil)
+        searchController.searchBar.placeholder = "원하는 콘텐츠를 검색하세요."
+        return searchController
+    }()
+    var mainView: UIView = {
+        let view = UIView()
+        view.backgroundColor = .systemRed
+        return view
+    }()
+    var filterView: UIView = {
+        let view = UIView()
+        view.backgroundColor = .white
+        return view
+    }()
+    var totalLabel: UILabel = {
+        let label = UILabel()
+        label.text = "전체 0"
+        return label
+    }()
+    var changeShowButton: UIButton = {
+        let button = UIButton()
+        button.setTitle("B", for: .normal)
+        button.backgroundColor = UIColor.blue
+        return button
+    }()
+    var sortButton: UIButton = {
+        let button = UIButton()
+        button.setTitle("최근순", for: .normal)
+        button.backgroundColor = UIColor.blue
+        return button
+    }()
+    
+    var filterCollectionView: UICollectionView!
+    var contentsCollectionView: UICollectionView!
```

Comment:
p2;
@beansbin 💜
혹시 여기도 !없이 
```suggestion
    var filterCollectionView = UICollectionView()
    var contentsCollectionView = UICollectionView()
```
요렇게 해주세요..💜 요것만 바꾸면 될 거 같어라

Reply:
수정 완료 했습니다! ❤️